homeassistant/components/aemet/abstract_aemet_sensor.py

```diff
@@ -0,0 +1,77 @@
+"""Abstraction form AEMET OpenData sensors."""
+from homeassistant.const import ATTR_ATTRIBUTION
+from homeassistant.helpers.entity import Entity
+
+from .const import ATTRIBUTION, SENSOR_DEVICE_CLASS, SENSOR_NAME, SENSOR_UNIT
+from .weather_update_coordinator import WeatherUpdateCoordinator
+
+
+class AbstractAemetSensor(Entity):
```

Comment:
Why aren't we inheriting `CoordinatorEntity`?

Reply:
As above, this is inherited from OpenWeatherMap: https://github.com/home-assistant/core/blob/fd177441b3cfddb22b893b26e91862307bbcc437/homeassistant/components/openweathermap/abstract_owm_sensor.py#L9